cogito/put_private_test.go

```diff
@@ -264,11 +292,13 @@ func TestParseGitPseudoURLFailure(t *testing.T) {
 			wantErr: "invalid git SSH URL git@github.com/Pix4D/cogito.git: want exactly one ':'",
 		},
 		{
+			// currently failing due to the new safeParse method returning only the error
```

Comment:
```
    --- FAIL: TestParseGitPseudoURLFailure/invalid_HTTPS_URL (0.00s)
        put_private_test.go:327: assertion failed: expected error "parse \"https://github.com:Pix4D/cogito.git\": invalid port \":Pix4D\" after host", got "invalid port \":Pix4D\" after host"
    --- FAIL: TestParseGitPseudoURLFailure/invalid_HTTP_URL (0.00s)
        put_private_test.go:327: assertion failed: expected error "parse \"http://github.com:Pix4D/cogito.git\": invalid port \":Pix4D\" after host", got "invalid port \":Pix4D\" after host"
```

Reply:
I think that having only the error should be enough, in this case `invalid port` is useful. The URL is in the pipeline configuration if we really need it.